helpers/control.py

```diff
@@ -492,23 +492,32 @@ def __servers_pool_size(n_servers):
     return min(n_servers, MAX_THREADS)
 
 def servers_start(servers):
-    threads = __servers_pool_size(len(servers))
```

Comment:
`__servers_pool_size()` is not used any more. The original code was intended to speedup test environment deployment. Since the feature is not required any more, unneeded code should be removed.

Reply:
fixed